solr/CHANGES.txt

```diff
@@ -37,6 +37,11 @@ New Features
 
 * SOLR-16500: Added --enable-preview for JDK 19 to enable MemorySegment support in MMapDirectory (Kevin Risden)
 
+Bug Fixes
```

Comment:
There is already a section for bug fixes

Reply:
Thanks  :)